UT4MasterServer/Controllers/Epic/PersonaController.cs

```diff
@@ -124,6 +124,8 @@ public async Task<IActionResult> GetAccount(string id)
 			return NotFound();
 		}
 
+		account.CountryFlag = account.CountryFlag.Replace('.', ' ');
```

Comment:
Same here. This should be done on frontend.

Reply:
Replaced on FE.